src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const BIG_DISCOUNT = 50;
+  const SMALL_DISCOUNT = 20;
+
+  const PERIOD_WITHOUT_DISCOUNT = 2;
```

Comment:
The constant 'PERIOD_WITHOUT_DISCOUNT' is not necessary as its value is not used in a meaningful way. You can remove this constant and adjust the logic accordingly.